index.html

```diff
@@ -9,18 +9,32 @@
         <script src="js/fruits_and_veggies_test.js"></script>
         <script src="js/app.js"></script>
     </head>
-    <body>
+    <body ng-app="app" ng-controller="ctrl">
         <div class="container">
             <h1>Fruit and Veggies</h1>
```

Comment:
Here is my CSS for red and green on win or fail:

    .has-success {
        background-color: green;
    }

    .has-fail {
        background-color: red;
    }